app/views_redesigned/properties/_tr.html.erb

```diff
@@ -0,0 +1,65 @@
+<%= tag.tr class: classes("even": property_counter.even?, "odd": property_counter.odd?, "opacity-40": %w[archived blacklisted rejected].any?(ENUMS::PROPERTY_STATUSES[property.status])) do %>
+
+  <td class="align-middle">
+    <%= image_tag property.favicon_image_url %>
+  </td>
+
+  <td class="align-middle">
+    <%= link_to property.name, property, title: "View Property", data: tooltip_expando %><br/>
+    <small><%= link_to truncate(property.display_url, length: 30), property.url, target: "_blank" %></small>
+  </td>
+
+  <% if authorized_user.can_admin_system? %>
+    <td class="align-middle">
+      <%= link_to property.user.full_name, user_path(property.user), title: "View User", data: tooltip_expando %>
+      <% if property.user.organization %>
+        <br/>
+        <small><%= link_to property.user.organization.name, property.user.organization, class: "text-dark" %></small>
+      <% end %>
+    </td>
+  <% end %>
+
+  <td class="align-middle" style="font-size: 12px;">
+    <% if property.restrict_to_sponsor_campaigns? %>
+      <span class="opacity-30">N/A</span>
+    <% else %>
+      <span class="text-muted">D:</span> <%= number_to_human property.property_traffic_estimates.first&.visitors_daily, units: { thousand: "K", million: "M" } %><br/>
+      <span class="text-muted">M:</span> <%= number_to_human property.property_traffic_estimates.first&.visitors_monthly, units: { thousand: "K", million: "M" } %><br/>
+      <span class="text-muted">Y:</span> <%= number_to_human property.property_traffic_estimates.first&.visitors_yearly, units: { thousand: "K", million: "M" } %><br/>
+    <% end %>
+  </td>
+
+  <td class="align-middle text-muted">
+    <small class="d-block"><%= property.created_at.strftime("%d %b, %Y") %></small>
+    <small class="d-block"><%= property.created_at.strftime("%I:%M %p")  %></small>
+  </td>
+
+  <td class="align-middle text-<%= property_status_color(property.status) %>">
+    <span class="fas fa-circle small mr-1"></span>
+    <%= property.status.humanize %>
+  </td>
+
+  <td>
+    <% if property.restrict_to_sponsor_campaigns? %>
+      <span class="opacity-30">N/A</span>
+    <% else %>
+      <%= render_later async_property_sparkline_path(property_id: property.id) do %>
+        <i class="fa fa-spinner fa-spin text-primary"></i>
+      <% end %>
+    <% end %>
+  </td>
+
+  <td class="align-middle text-muted text-right">
+    <% if property.restrict_to_sponsor_campaigns? %>
+      <span class="opacity-30">N/A</span>
+    <% else %>
+      <%= render_later async_property_click_rate_path(property_id: property.id) do %>
+        <i class="fa fa-spinner fa-spin text-primary"></i>
+      <% end %>
+    <% end %>
+  </td>
+
+  <td class="align-middle text-right text-secondary">
+    <%= render "/@shared/table_buttons/edit", link: edit_property_path(property) %>
```

Comment:
This should be:

```suggestion
    <%= render "/shared/table_buttons/edit", link: edit_property_path(property) %>
```

but let's wait until [this](https://github.com/gitcoinco/code_fund_ads/pull/729/files#diff-ec3a97e817fce8b2ade68f6a7c5c7687R13) method is added and use it instead.